internal/app/storage/storage.go

```diff
@@ -0,0 +1,35 @@
+package storage
+
+//type ObjStorage struct {
+//	shortUrl string
+//}
+
+type Storage map[string]string
+
+func NewStorage() Storage {
+	arr := make(Storage)
+	return arr
+}
+
+func (strg Storage) Save(newUrl string, oldUrl string) string {
```

Comment:
Стораджу не важно что с точки зрения бизнес логики мы сюда складываем. Возможно захочется использовать этот пакет в другом месте кода и хранить что-то совершенно другое
```suggestion
func (strg Storage) Save(key, value string) string {
```